src/pymovements/events/idt.py

```diff
@@ -0,0 +1,118 @@
+"""
+This module holds the implementation for idt algorithm.
+"""
+from __future__ import annotations
+
+import numpy as np
+
+from pymovements.events import Fixation
+
+
+def dispersion(positions: list[list[float]] | np.ndarray) -> float:
+    """
+    Compute the dispersion of a group of consecutive points in a 2D position time series.
+
+    The dispersion is defined as the sum of the differences between
+    the points' maximum and minimum x and y values
+
+    Parameters
+    ----------
+    positions: array-like
+        Continuous 2D position time series.
+
+    Returns
+    -------
+    dispersion: float
+        Dispersion of the group of points.
+    """
+    return sum(np.max(positions, axis=0) - np.min(positions, axis=0))
+
+
+def idt(
+        positions: list[list[float]] | np.ndarray,
+        dispersion_threshold: float,
+        duration_threshold: int,
+) -> list[Fixation]:
+    """
+    Fixation identification based on dispersion threshold.
+
+    The implementation is based on the description and pseudocode
+    from Salvucci and Goldberg :cite:p:`SalvucciGoldberg2000`.
+
+    Parameters
+    ----------
+    positions: array-like, shape (N, 2)
+        Continuous 2D position time series
+    dispersion_threshold: float
+        Threshold for dispersion for a group of consecutive samples to be identified as fixation
+    duration_threshold: int
+        Minimum fixation duration in number of samples
+
+    Returns
+    -------
+    list[Fixation]:
+        List of Fixation events
+
+    Raises
+    ------
+    ValueError
+        If positions is not shaped (N, 2)
+        If dispersion_threshold is not greater than 0
+        If duration_threshold is not greater than 0
+    """
+
+    positions = np.array(positions)
+
+    # make sure x is 2d
+    if positions.ndim != 2:
+        raise ValueError(
```

Comment:
if possible to stay below 100 characters, use a single line for all these ValueErrors